darts/models/forecasting/ensemble_model.py

```diff
@@ -266,13 +270,21 @@ def predict(
             future_covariates=future_covariates,
             num_samples=num_samples,
             verbose=verbose,
+            predict_likelihood_parameters=predict_likelihood_parameters,
         )
 
-        # for multi-level models, forecasting models can generate arbitrary number of samples
+        if series is None:
```

Comment:
were these changes required?

Reply:
Yes, because of the changes to support multivariate ts with the `NaiveEnsemble`, the argument `series` of `ensemble()` is not optional anymore. But I can move this to `NaiveEnsemble.ensemble()` if you prefer